OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -47,28 +58,59 @@ class MissileInfo : IProjectileInfo
 		[Desc("Probability of locking onto and following target.")]
 		public readonly int LockOnProbability = 100;
 
-		[Desc("In n/256 per tick.")]
-		public readonly int RateOfTurn = 5;
+		[Desc("Horizontal rate of turn.")]
+		public readonly int RateOfTurn = 5; // Rename to HorizontalRateOfTurn
```

Comment:
Nit: Add a 'TODO:' before Rename, easier to keep track of these if you have a buzzword to search for.